packages/strapi-parts/src/Tabs/components.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import { Box } from '../Box';
+import { Row } from '../Row';
+
+/** Simple variant */
+export const SimpleTabBox = styled(Box)`
+  border-bottom: 2px solid
+    ${({ theme, selected, hasError }) => {
+      if (selected) {
+        if (hasError) {
+          return theme.colors.danger600;
```

Comment:
The error color is displayed only if the tab is selected?

Reply:
The text color, in error, is always red. However, the border should be red and visible only when the tab is selected, according to figma 😊